src/content/docs/en/developers/verifying-smart-contracts.mdx

```diff
@@ -5,31 +5,39 @@ title: "Verifying Smart Contracts"
 lang: "en"
 permalink: "developers/verifying-smart-contracts"
 whatsnext: { "Scroll Contracts": "/developers/scroll-contracts" }
-excerpt: "Easily verify your smart contracts with Scroll-supported developer tooling or the Blockscout Web API"
+excerpt: "Easily verify your smart contracts with Scroll-supported developer tooling"
 ---
 
 import Aside from "../../../../components/Aside.astro"
 import ClickToZoom from "../../../../components/ClickToZoom.astro"
 import verify1 from "./_images/verify1.png"
 import CodeSample from "../../../../components/CodeSample/CodeSample.astro"
 
-After deploying your smart contracts, it's important to verify your code on [our block explorer](https://scrollscan.com/) or the [Sepolia block explorer](https://sepolia-blockscout.scroll.io).
+After deploying your smart contracts, it's important to verify your code on our block explorers. This can be done in an automated way using your developer tooling or the Web UI.
 
-This can be done in an automated way using your developer tooling or using Blockscout's Web UI.
-
-<Aside type="tip" title="Blockscout vs Etherscan">
+{/* <Aside type="tip" title="Blockscout vs Etherscan">
   The below instructions for verifying on our Sepolia Blockscout instance will need to be altered to work with Etherscan
   deployments. As additional services come online for mainnet, we'll update these instructions.
-</Aside>
+</Aside> */}
 
 ## Using Developer Tools
 
-Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, and it's straightforward to use these tools with the Scroll Sepolia Testnet.
+Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, so it's straightforward to use these tools using the APIs of either of these block explorers.
+
+| Network    | Scroll Scroll                    | Sepolia                                  |
+| ---------- | -------------------------------- | ---------------------------------------- |
+| Scrollscan | https://api.scrollscan.com/api   | https://api-sepolia.scrollscan.com/api   |
+| Blockscout | https://blockscout.scroll.io/api | https://sepolia-blockscout.scroll.io/api |
```

Comment:
Do we even want to emphasize Blockscout? For the most part we consider it an internal tool and don't want developers defaulting to using it for contract verification.

Reply:
Actually, for now let's leave it.